vertical-tabbrowser.xml

```diff
@@ -220,6 +191,21 @@
           draggedTab.style.transform = 'translateY(' + translateVal + 'px)';
           // console.log('draggedTab transform: ' + draggedTab.style.transform + ' should be ' + 'translateY(' + translateVal + 'px)');
 
+          let tabbrowser = document.getElementById('tabbrowser-tabs');
+          let scrollbox = document.getAnonymousElementByAttribute(tabbrowser, 'anonid', 'arrowscrollbox');
+          let scrollbuttonDown = document.getAnonymousElementByAttribute(scrollbox, 'anonid', 'scrollbutton-down');
+          let scrollbuttonUp = document.getAnonymousElementByAttribute(scrollbox, 'anonid', 'scrollbutton-up');
+          let mouseY = event.clientY;
+
+          if (mouseY > (document.getElementById('tabbrowser-tabs').boxObject.height + 10)){
+            scrollbuttonDown.onmousedown(event);
+          } else if (mouseY < 65) {
```

Comment:
So, I feel like this is too far to the top…  On the bottom, it scrolls when it gets close to the last tab, but on the top it only scrolls when it's over the toolbar.  And I feel like it should scroll when you're over the first tab as well, because you'll want to drop the tab in the tab list, not on the toolbar.  (Like, the scrolling parts should be consistent in the tab list on the top and bottom, with the toolbar being an extra bonus area, if that makes any sense.)